backend/routes.js

```diff
@@ -130,6 +170,26 @@ export default class Server {
             file: 'oauth.html'
         }));
 
+        server.get(/\/login\/?$/, restify.serveStatic({
+            directory: `${__dirname}/../static`,
+            file: 'login.html'
+        }));
+
+        server.post('/approved/:username', function hasAuth(req, res, next) {
```

Comment:
what if this was `/settings/users`? then, if we needed to expose other settings, we could add them with similar URL structures